bsef.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="fr-FR" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Bulletin de la Société Entomologique de France</title>
+    <title-short>bsef</title-short>
+    <id>http://www.zotero.org/styles/bsef</id>
+    <link href="http://www.zotero.org/styles/bsef" rel="self"/>
+    <link href="http://www.lasef.org/publications/le-bulletin-de-la-sef/" rel="documentation"/>
```

Comment:
I began with bmj not to start from nothing, but it's so far now of it than deleted the template line. Must I still mention it?